src/resources/postcss/views/month/_calendar-event.pcss

```diff
@@ -37,16 +41,8 @@
 		font-weight: var(--font-weight-regular);
 	}
 
-	.tribe-events-calendar-month__calendar-event-tooltip {
-		background-color: var(--color-background);
-		border: 1px solid var(--color-border-default);
-		box-shadow: var(--box-shadow-tooltip);
-		display: none;
-		left: calc(var(--spacer-7) * -1);
-		padding: var(--spacer-4);
-		position: absolute;
-		top: calc(100% + var(--spacer-1));
-		width: calc(100% + (2 * var(--spacer-7)));
+	.tribe-events-calendar-month__calendar-event-title-link {
+		color: var(--color-text-primary);
```

Comment:
I'm not sure if I follow you on this one. I'd be all in for setting the color on `.tribe-common a`, otherwise on classes like this one. If not, any theme will end up having their own `a` color style. And the idea would be to be in control of the link color for the calendar event title, right?

Reply:
Yup, `.tribe-common a would be the way to go. We want to override user agent and theme styles.